GTK+ 3.x allows for multiple target backends (e.g. x11 and broadway)

--- origsrc/devhelp-3.2.0/acinclude.m4	2011-08-21 05:00:05.000000000 -0500
+++ src/devhelp-3.2.0/acinclude.m4	2011-11-17 18:08:31.105586600 -0600
@@ -1,5 +1,5 @@
 AC_DEFUN([IGE_PLATFORM_CHECK],[
-    gdk_target=`$PKG_CONFIG --variable=targets gtk+-3.0`
+    for gdk_target in `$PKG_CONFIG --variable=targets gtk+-3.0`; do break; done
 
     if test "x$gdk_target" = "xquartz"; then
         AC_MSG_CHECKING([checking for Mac OS X support])
